#! /bin/sh /usr/share/dpatch/dpatch-run
## 12_missing_malloc.dpatch by Mario Limonciello <superm1@ubuntu.com>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Add missing malloc includes needed for G++ 4.3.1

@DPATCH@
diff -urNad mythplugins-0.21.0+fixes17416~/mythmusic/mythmusic/aacdecoder.cpp mythplugins-0.21.0+fixes17416/mythmusic/mythmusic/aacdecoder.cpp
--- mythplugins-0.21.0+fixes17416~/mythmusic/mythmusic/aacdecoder.cpp	2008-06-01 18:16:43.000000000 -0500
+++ mythplugins-0.21.0+fixes17416/mythmusic/mythmusic/aacdecoder.cpp	2008-06-03 08:40:20.000000000 -0500
@@ -19,6 +19,7 @@
 #include <qobject.h>
 #include <qiodevice.h>
 #include <qfile.h>
+#include <stdlib.h>
 
 #include "aacdecoder.h"
 #include "constants.h"
diff -urNad mythplugins-0.21.0+fixes17416~/mythmusic/mythmusic/metaiomp4.cpp mythplugins-0.21.0+fixes17416/mythmusic/mythmusic/metaiomp4.cpp
--- mythplugins-0.21.0+fixes17416~/mythmusic/mythmusic/metaiomp4.cpp	2006-08-21 21:06:03.000000000 -0500
+++ mythplugins-0.21.0+fixes17416/mythmusic/mythmusic/metaiomp4.cpp	2008-06-03 08:40:37.000000000 -0500
@@ -3,6 +3,7 @@
 #include <fcntl.h>
 #include <unistd.h>
 #include <sys/types.h>
+#include <stdlib.h>
 
 using namespace std;
 
